src/lib/components/specialized/services/display/service-key-informations.svelte

```diff
@@ -19,6 +19,18 @@
 
   export let service: Service;
   export let servicesOptions: ServicesOptions;
+
+  // trier les types dans l'ordre d'affichage du formulaire
+  function sortServiceTypes(st1: string, st2: string) {
+    const order = ["En autonomie", "Accompagnement", "Atelier", "Délégation"];
+    // pas de ternaires imbriqués : c'est pourtant plus lisible et plus court
+    // eslint-disable-next-line no-nested-ternary
+    return order.indexOf(st1) > order.indexOf(st2)
+      ? 1
+      : order.indexOf(st1) < order.indexOf(st2)
+        ? -1
+        : 0;
+  }
```

Comment:
Cela me paraît bien compliqué et je ne comprends pas bien cette liste `order`.

Si on veut juste trier dans l'ordre alphabétique comme dans le formulaire, il suffirait de faire ça :

```suggestion
  $: sortedServiceKindsDisplay = service.kindsDisplay.sort((a, b) => a.localeCompare(b));
```

Reply:
Me doutais que c'était un peu alambiqué. Merci

Reply:
Bon ben, c'est moi qui avais une vision partielle de la carte. C'est un simple tri alpha, donc pas besoin de tout le tintouin